git-repo-prep/static/repo-settings/.github/settings.yml

```diff
@@ -0,0 +1,45 @@
+# These settings are synced to GitHub by https://probot.github.io/apps/settings/
+
+repository:
+  # See https://docs.github.com/en/rest/reference/repos#update-a-repository for all available settings.
+  private: false
+
+  has_issues: false
+  has_wiki: false
+  has_projects: false
+  has_downloads: false
+
+  default_branch: main
```

Comment:
Does this settings.yml file contain all of the options that the probot settings plugin understands? (just checking. Hopefully it does)

I'm torn... It'd be nice to see which of these options are defaults and which were changed by humans. I guess git blame can help point that out. hmm.

Reply:
I did not include the `labels` or `milestones` configuration sections.

The `repository` section is missing these:
```yml
  allow_squash_merge: true
  allow_merge_commit: true
  allow_rebase_merge: true

  enable_automated_security_fixes: false
  enable_vulnerability_alerts: false

  topics: github, probot
  homepage: ...
  name: ...
```

Other than that, everything that I know of that could be included is included.